### ISSUE TITLE
[Quedtion] wann wird eine schedule gelöscht?

### ISSUE THREAD

**Issue by BenAhrdt** [ioBroker/ioBroker.javascript#1327]
Wenn ich eine schedule erzeuge mit
`myschedule = schedule(….)`
Wird diese dann auch beim späteren Speichern und restart des Skript gelöscht?

ich hatte nämlich den Fall, dass eine erzeugte schedule nicht aufgerufen wurde.
Es könnte aber sein, dass ich danach nochmal das Skript neu gestartet habe.
Müsste man diese dann beim Start wieder zuweisen?
(Wird bei Änderungen in der Visu erzeugt)

**Comment by BenAhrdt** [comment]
Wenn ich eine schedule in einem
Skript definiere…. Wird diese doch dann gecleart, wen das Script gestoppt wird, oder?

**Comment by Apollon77** [comment]
Ja, das habe ich oben geschrieben. Das erfolgt i normalfall automatisch

**Comment by BenAhrdt** [comment]
Ok. Dann wird das vielleicht der Grund gewesen sein, warum die schedule nicht ausgeführt wurde, wenn die anhand der visuangabe gema hat wurde und dann das Skript neu gestartet wurde. Muss da dann nochmal abfangen.

**Comment by Apollon77** [comment]
aber an sich wird über die "cisuanlage" auch nur quellcode erzeugt
